ajax/libs/iphone-inline-video/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "iphone-inline-video",
+  "filename": "iphone-inline-video.min.js",
+  "version": "2.2.2",
+  "description": "Make videos playable inline on the iPhone (prevents automatic fullscreen)",
+  "homepage": "https://npm.im/iphone-inline-video",
```

Comment:
This link always redirct to npmjs page.
@bfred-it Would you give us some suggestion about the homepage?
Which link do you like? `http://bfred-it.github.io/iphone-inline-video/` or `https://npm.im/iphone-inline-video`?

Many thanks.

Reply:
The GitHub repo or none. I use npm.im for a quick link to NPM

Reply:
@bfred-it Thank you.
We don't use GitHub repo as homepage because we have `repository` info.

Maybe we can use `http://bfred-it.github.io/iphone-inline-video/` as the homepage.
What do you think?
Because this page can let people know more demo of lib.

Many thanks.

Reply:
I think either repo or nothing

Reply:
Thank you.

@sashberd Let's use `http://bfred-it.github.io/iphone-inline-video/` as the homepage, what do you think?
Many thanks.